leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,8 @@
   <!-- To store the heap dumps and leak analysis results. -->
   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+  <!-- To allow starting foreground services on Android P+ -->
```

Comment:
Can you update and add the link to https://developer.android.com/preview/behavior-changes#fg-svc in this comment?